pkg/login/social/generic_oauth.go

```diff
@@ -239,6 +249,29 @@ func (s *SocialGenericOAuth) extractRole(data *UserInfoJson) (string, error) {
 	return role, nil
 }
 
+func (s *SocialGenericOAuth) extractGroupMappings(data *UserInfoJson) ([]setting.OAuthGroupMapping, error) {
+	var groupMappings []setting.OAuthGroupMapping
+	if s.groupMappings == nil {
```

Comment:
Given my comment regarding default init in social.go we could remove this if statement.

Reply:
Removed this condition